### PR TITLE
fix(CMake): add a download target to only download all third parties

### PR DESCRIPTION
**Pull request by BotellaA**

### CODE REVIEW

File: cmake/SuperBuild.cmake [hunk 1]
```diff
@@ -18,6 +18,8 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+cmake_policy(SET CMP0114 NEW)
```
Comment (panquez) [inline cmake/SuperBuild.cmake:21]: Only for personal culture, what does it stand for?

**Review by panquez — APPROVED** [review]

File: cmake/SuperBuild.cmake (hunk 1 above)
Comment (BotellaA) [reply cmake/SuperBuild.cmake:21]: https://cmake.org/cmake/help/latest/policy/CMP0114.html
mainly a change in the behavior of the dependent steps between several external projects

**Pull request merged**